pwndbg/commands/procinfo.py

```diff
@@ -211,6 +223,10 @@ def procinfo() -> None:
     if not proc.status:
         return
 
+    # print("%-10s %s" % ("cmdline", proc.something))
```

Comment:
for some reason when i try to use proc.cmdline i get an error message like this:

```
pwndbg> procinfo
exe        '/usr/bin/ls'
Python Exception <class 'FileNotFoundError'>: [Errno 2] No such file or directory: '/home/dan/pwndbg/pwndbg/../requirements.txt'
Error occurred in Python: [Errno 2] No such file or directory: '/home/dan/pwndbg/pwndbg/../requirements.txt'
```

( yes the comment shows proc.something my bad but i was testing with proc.cmdline )

Reply:
This comes from https://github.com/pwndbg/pwndbg/issues/1865 - we need to fix this ASAP :/